docs/src/man/split_apply_combine.md

```diff
@@ -124,6 +126,54 @@ julia> by(iris, :Species,
 │ 2   │ Iris-versicolor │ 0.717655 │ 213.0   │
 │ 3   │ Iris-virginica  │ 0.842744 │ 277.6   │
 
+julia> by(iris, :Species,
+          :PetalLength, :SepalLength,
+          AsTable([:PetalLength, :SepalLength]) => sum)
```

Comment:
Here too, this example doesn't sound very useful as grouping has no effect. Not sure what kind of example could be useful, but if we can't find one maybe better not show anything (showing it in `select` can be enough). Same below.

Reply:
ok - changed both.